lib/ensemble_app.dart

```diff
@@ -117,10 +117,38 @@ class EnsembleAppState extends State<EnsembleApp> with WidgetsBindingObserver {
       Workmanager().initialize(callbackDispatcher, isInDebugMode: false);
       initDeepLink(AppLifecycleState.resumed);
     }
+
+    WidgetsBinding.instance.addPostFrameCallback((_) async {
+      List<int?> functionIds = [];
+      final callbacks = Ensemble().getCallbacksAfterInitialization();
+      for (final Map<String, Object?> callback in callbacks) {
+        final id = callback['id'] as int?;
+        final method = callback['method'] as Function?;
+        final positionalPayloads = callback['positionalArgs'] as List<dynamic>?;
+        final namedPayloads = callback['namedArgs'] as Map<String, dynamic>?;
+        Map<Symbol, dynamic>? namedParams;
+        namedPayloads?.forEach((key, value) {
+          namedParams = {Symbol(key): value};
+        });
+
+        if (method != null) {
+          await Function.apply(method, positionalPayloads, namedParams);
+          functionIds.add(id);
+        }
+      }
+
+      // Looping function id to remove the functions in the callback object with the id
+      for (final id in functionIds) {
+        callbacks.removeWhere((element) => element['id'] == id);
```

Comment:
Please read my original comment for this task. Either you put function execution inside try/catch, or you remove the item then execute (with a try catch). This way if the function throws an exception, our app is not hosed permanently. Remmeber this is what the app will do on start-up - a bad function will mess the app, regardless if you restart the app or not.